docs/api/session.md

```diff
@@ -1513,6 +1513,22 @@ is emitted.
 Returns `string | null` - The absolute file system path where data for this
 session is persisted on disk.  For in memory sessions this returns `null`.
 
+#### `ses.clearBrowsingData()`
```

Comment:
API LGTM

On the documentation side, we should also explicitly note how it differs from `ses.clearStorageData` as per Cheng's review comment above.